.cirrus.yml

```diff
@@ -16,26 +16,38 @@ env:
   CTIMETEST: yes
   BENCH: yes
   ITERS: 2
-  # We only need the top commit
-  CIRRUS_CLONE_DEPTH: 1
+  MAKEFLAGS: -j2
```

Comment:
What does this do?

Reply:
`MAKEFLAGS` is a way to set flags for `make` globally. `make` will pick this up. See in 5.7.3 ian the manual (https://www.gnu.org/software/make/manual/make.html#index-MAKEFLAGS) or in Appendix A:

> The MAKEFLAGS variable can also be useful if you want to have certain options, such as ‘-k’ (see Summary of Options), set each time you run make. You simply put a value for MAKEFLAGS in your environment.

Reply:
ugh but now I need to remove `-j2`  from the invocations in the script... Sorry, I screwed up with too many fixups  and squashes

Reply:
Ah appendix A is helpful. The first sentence in the manual is "Flags [...] are passed automatically to the sub-make through the variable MAKEFLAGS." which would make this use of MAKEFLAGS rather esoteric.

Reply:
Yeah, I admit the docs are somewhat obscure. (Maybe this was introduced as a way to pass stuff to sub-makes but it's widely used to specify -j2 for example, and there's nothing wrong with that.) See also POSIX: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html

I'll update once again to remove the explicit flags from the command line.

Reply:
fixed